keps/sig-node/4205-psi-metric/README.md

```diff
@@ -314,32 +264,25 @@ https://storage.googleapis.com/k8s-triage/index.html
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
 -->
 
-- <test>: <link to test coverage>
+- `test/e2e_node/summary_test.go`: `https://storage.googleapis.com/k8s-triage/index.html?test=test%2Fe2e_node%2Fsummary_test.go`
 
 ### Graduation Criteria
 
-#### Phase 1: Alpha
+#### Alpha
 
 - PSI integrated in kubelet behind a feature flag.
 - Unit tests to check the fields are populated in the 
   Summary API response.
 
-#### Phase 2: Alpha
-
-- Implement Phase 2 of the enhancement which enables kubelet to 
-report node conditions based off PSI values.
-- Initial e2e tests completed and enabled if CRI implementation supports
-it.
-- Add documentation for the feature.
-
 #### Beta
 
 - Feature gate is enabled by default.
 - Extend e2e test coverage.
 - Allowing time for feedback.
```

Comment:
Have we gathered any performance implications of enabling and gather PSI? Will there be a knob for turning this on/off?

Reply:
no knob once this goes GA, though I do agree there should be perf testing done. When CRI stats comes along, technically the CRI implementation could be configured whether to report PSI metrics or not. We also could introduce a knob in the kubelet config, as CRI stats has been "coming" for years now...

Reply:
another thing to consider--if the node has psi off but cadvisor is trying to collect the metrics, what happens?

Reply:
I added performance testing to the beta requirements. PTAL

Reply:
@roycaihw I don't see it in the beta list above.

Reply:
Sorry somehow I failed to push the change :( Thanks for fixing it!